vlasovsolver/cpu_trans_map_amr.cpp

```diff
@@ -1206,7 +1247,7 @@ bool trans_map_1d_amr(const dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>&
 
                         // Unpack the vector data
                         Realf vector[VECL];
```

Comment:
Question @ursg @galfthan: do you expect a performance gain if this declaration was moved outside? SImilarly, there's a bunch of ints that get redeclared inside loops at every iteration. Is the compiler clever enough for these? Or is it less of an issue for small data types?

Reply:
Yeah, the compiler should be clever enough. And even if not: declaration of an automatic variable like that on the stack does not actually go through the heap allocator, but just shifts the stack pointer by a couple of bytes. So this is literally only a single integer addition and does not even perform a memory access. Don't worry about it.